honeybadger/honeybadger.go

```diff
@@ -0,0 +1 @@
+package hb
```

Comment:
plz remove this dir, HoneyBadger component will be placed in root of this project!

Reply:
@boohyunsik